src/styles/watch.scss

```
@@ -0,0 +1,62 @@
+@import "./variables";
+
+.watch {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 600px;
+  height: 600px;
+  border: 10px solid;
+  border-radius: 50%;
```

Роби відступи між різними групами правил, так їх простіше читати і підтримувати
```suggestion
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  
  width: 600px;
  height: 600px;
  
  border: 10px solid;
  border-radius: 50%;
```